plugins/http2_adapter/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ See the [Migration Guide][] for the complete breaking changes list.**
 
 *None.*
 
+## 2.7.0
```

Comment:
We don't update the version header in changes. Please refer to other PRs and see how they handle this.